src/components/navigation-header/components/dropdown-menu/dropdown-menu.module.css

```diff
@@ -6,6 +6,7 @@
 .activatorWrapper {
   padding-bottom: 16px;
   padding-top: 16px;
+  margin-right: 12px;
```

Comment:
I moved this spacing to the dropdown component, since they always render together, but the product logo can render alone.